maxValue/mapper.py

```diff
@@ -1,13 +1,11 @@
 import sys
-import csv
 
```

Comment:
I think a better name for this directory would be something like `highest_sale` because `maxValue` tells nothing about the task itself.